content/blog/2021/03/2021-03-01-update-center-certificate-rotation.adoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "Update-Center certificate rotation"
+tags:
+- announcement
+- update-center
+author: olblak
+description: Announcement, Update.jenkins.io certificate rotation
+opengraph:
+  image: /images/post-images/2021-03-01-update-center-certificate-rotation/certificate-rotation-opengraph.png
+---
+
+= Update-center Certificate Rotation
+
+**IMPORTANT**
```

Comment:
```suggestion

```

Reply:
Included in 18d9b2cb920c3a60856cc5e20dcb67ef31d0ea28